canary/Models/Connectathon.cs

```diff
@@ -10,32 +10,30 @@ public class Connectathon
         public static string fhirVersion = "R4";    // used to generate files
         public Connectathon() { }
 
-        public static DeathRecord FromId(int id, string state = null)
+        public static DeathRecord FromId(int id, int? certificateNumber = null, string state = null)
         {
             DeathRecord record = null;
             switch (id)
             {
                 case 1:
-                    record = JanetPage();
+                    record = JanetPage(certificateNumber);
                     break;
                 case 2:
-                    record = MadelynPatel();
+                    record = MadelynPatel(certificateNumber);
                     break;
                 case 3:
-                    record = VivienneLeeWright();
+                    record = VivienneLeeWright(certificateNumber);
                     break;
                 case 4:
-                    record = JavierLuisPerez(partialRecord: false);
+                    record = JavierLuisPerez(certificateNumber, partialRecord: false);
                     break;
                 case 5:
-                    record = JavierLuisPerez(partialRecord: true);
+                    record = JavierLuisPerez(certificateNumber, partialRecord: true);
```

Comment:
It may be slightly `DRY`'er to handle this the same way state's are handled, by checking at the end and overriding the `Identifier` if a certificateNumber was provided.

Reply:
Done!